modules/jaggery-apps/portal/gadgets/connected_accounts/controllers/service-clients/identityProviderMgtClient.jag

```diff
@@ -3,20 +3,21 @@
 var log = new Log();
```

Comment:
When defining log object in jagger, its better to define it with the file path. Then, we know which jaggery file printed that log. For example,

var log = new Log("site/pages/index.jag").

Make this a practice for all the jaggery codes.